### DISCUSSION TITLE
filter queries not including symbols and whitespace [Feb 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5242] [Discord Threads]
> 🙋‍♂️ **kyleecodes**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Joecode**  
#### Thanks <@446452643221602304> . I see, so is it correct that the type of column determines which characters are allowed to be input?  
#  
> 🙋‍♂️ **kyleecodes**  
#### Correct, column type determines the data type allowed to be input  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1067193074989666354)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
